workflows/taxprofiler.nf

```diff
@@ -40,7 +40,7 @@ include { INPUT_CHECK         } from '../subworkflows/local/input_check'
 
 include { DB_CHECK            } from '../subworkflows/local/db_check'
 include { FASTQ_PREPROCESSING } from '../subworkflows/local/preprocessing'
```

Comment:
Could you rename/update FASTQ_PREPROCESSING and the file itself to SHORTREAD_PREPROCESSING for consistency?

Reply:
Yes, they are updated now.